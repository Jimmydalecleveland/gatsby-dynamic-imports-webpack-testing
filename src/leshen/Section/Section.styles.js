import styled from "@emotion/styled"
import { css } from "@emotion/core"
import { stripUnit } from "polished"

import fluidRange from "../utils/fluidRange"
import { WidthContainer, ColorDeterminer } from "../utils/helpers.styles"

export const Section = styled.section(
  WidthContainer,
  ColorDeterminer,
  ({ theme }) => css`
    ${fluidRange("padding-top", [
      stripUnit(theme.spacing.lg),
      stripUnit(theme.spacing.max),
    ])};
    ${fluidRange("padding-bottom", [
      stripUnit(theme.spacing.lg),
      stripUnit(theme.spacing.max),
    ])};
    position: relative;
  `
)
