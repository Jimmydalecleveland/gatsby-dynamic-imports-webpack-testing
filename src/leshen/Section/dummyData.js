export const SVG_IMAGE_DATA = {
  svg: {
    code:
      '<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 288.325 288.325"><path fill="none" d="M0 0h288.325v288.325H0z"></path><circle cx="143.517" cy="143.204" r="116.335" fill="#2cd6ae"></circle><g fill="#21b790"><path d="M67.602 160.898h1.319v2.637h-1.319zM77.361 163.532h-2.109v-2.637h2.109zm-4.219 0h-2.11v-2.637h2.11zM79.471 160.898h2.637v2.637h-2.637zM82.108 158.219h-2.637v-2.678h2.637zm0-5.357h-2.637v-2.679h2.637zm0-5.358h-2.637v-2.679h2.637zm0-5.358h-2.637v-2.679h2.637zm0-5.358h-2.637v-2.681h2.637zm0-5.358h-2.637v-2.679h2.637zm0-5.358h-2.637v-2.679h2.637zm0-5.358h-2.637v-2.682h2.637zM79.471 112.719h2.637v2.637h-2.637zM134.799 115.355h-2.637v-2.638h2.637zm-5.269 0h-2.637v-2.638h2.637zm-5.269 0h-2.637v-2.638h2.637zm-5.269 0h-2.637v-2.638h2.637zm-5.27 0h-2.634v-2.638h2.634zm-5.269 0h-2.637v-2.638h2.637zm-5.268 0h-2.638v-2.638h2.636zm-5.269 0h-2.633v-2.638h2.638zm-5.269 0h-2.638v-2.638h2.638zm-5.27 0H84.74v-2.638h2.637zM137.434 112.719h1.319v2.637h-1.319z"></path></g><g fill="#21b790"><path d="M131.667 54.236h1.319v2.637h-1.319zM129.076 56.874h-2.593v-2.637h2.592zm-5.184 0h-2.591v-2.637h2.591zm-5.183 0h-2.591v-2.637h2.591zm-5.183 0h-2.592v-2.637h2.592zm-5.183 0h-2.592v-2.637h2.592zM101.841 54.236h1.319v2.637h-1.319z"></path></g><path d="M193.103 65.911h-46.05v-.135a14.094 14.094 0 011.871-28.053 19.591 19.591 0 0137.523-7.807 18.491 18.491 0 117.975 35.883v.095zm-44.372-2.637h44.011l.309-.017a15.851 15.851 0 10-6.831-30.433l-1.282.546-.474-1.312a16.953 16.953 0 00-32.9 5.747c0 .357.015.741.048 1.208l.109 1.56-1.556-.156a11.82 11.82 0 00-1.156-.058 11.457 11.457 0 00-.571 22.9z" fill="#21b790"></path><path d="M48.256 209.387a116.654 116.654 0 00192.31 0z" fill="#444"></path><g fill="#21b790"><path d="M251.798 208.332h-23l-.208-.072c-8-2.734-12.377-7.24-12.317-12.687v-88.755a12.759 12.759 0 0112.744-12.744h22.786a12.759 12.759 0 0112.744 12.744v88.772a12.758 12.758 0 01-12.749 12.742zm-22.562-2.637h22.562a10.119 10.119 0 0010.107-10.108v-88.769a10.119 10.119 0 00-10.107-10.107h-22.786a10.119 10.119 0 00-10.105 10.107v88.772c-.064 5.607 6.346 8.717 10.327 10.108z"></path><path d="M240.405 131.282a14.778 14.778 0 1114.778-14.779 14.8 14.8 0 01-14.778 14.779zm0-26.918a12.141 12.141 0 1012.14 12.141 12.155 12.155 0 00-12.14-12.14zM240.405 165.688a14.778 14.778 0 1114.778-14.779 14.8 14.8 0 01-14.778 14.779zm0-26.918a12.141 12.141 0 1012.14 12.142 12.155 12.155 0 00-12.14-12.14zM240.405 200.094a14.779 14.779 0 1114.777-14.78 14.8 14.8 0 01-14.777 14.78zm0-26.919a12.141 12.141 0 1012.14 12.143 12.157 12.157 0 00-12.14-12.141z"></path><path d="M240.404 126.465a9.916 9.916 0 119.916-9.916 9.915 9.915 0 01-9.916 9.916zm0-17.194a7.279 7.279 0 107.279 7.278 7.279 7.279 0 00-7.279-7.278z"></path><circle cx="240.405" cy="116.293" r="5.025"></circle></g><g fill="#21b790"><path d="M41.383 204.272c-15.333-.378-25.292-2.768-28.076-6.732a3.464 3.464 0 01-.6-3.106c.645-1.924 3.575-2.643 7.631-3.638a44.3 44.3 0 008.2-2.579c3.436-1.641 6.974-4.926 9.963-9.246a3.924 3.924 0 012.637-1.5v-.491a20.689 20.689 0 01-6.947-4.348c-5.391-5.135-18.826-24.53-16.292-52.252.79-8.632 3.81-20.3 13.713-26.84a24.393 24.393 0 0111.818-4.127h5.673a24.359 24.359 0 0111.848 4.128c5.467 3.611 12.285 11.2 13.713 26.84a71.547 71.547 0 01-5.73 35.964c-3.778 8.518-8.416 14.247-10.561 16.287a20.685 20.685 0 01-6.949 4.347v.489a3.861 3.861 0 012.609 1.464c3.018 4.361 6.557 7.646 9.993 9.288a44.346 44.346 0 008.2 2.578c4.057 1 6.988 1.714 7.631 3.639a3.464 3.464 0 01-.6 3.106c-2.784 3.964-12.743 6.354-28.043 6.731zm.869-24.255a2.442 2.442 0 00-1.55.41c-3.276 4.734-7.181 8.33-11.024 10.168a46.318 46.318 0 01-8.715 2.759c-2.133.523-5.355 1.313-5.776 1.954a1.145 1.145 0 00.278.714c.834 1.187 5.168 5.1 25.95 5.611h9.772c20.75-.511 25.082-4.423 25.918-5.61a1 1 0 00.259-.756c-.4-.6-3.624-1.39-5.757-1.912a46.216 46.216 0 01-8.714-2.76c-3.844-1.838-7.749-5.434-11-10.125a2.492 2.492 0 00-1.723-.452l-1.378.063-.006-5.074.986-.257a17.637 17.637 0 006.782-4.027c5.134-4.889 17.923-23.422 15.484-50.1-2.524-27.681-22.13-28.548-22.965-28.571h-5.59c-1.02.032-20.438 1.031-22.956 28.57-2.439 26.679 10.351 45.211 15.485 50.1a17.628 17.628 0 006.785 4.027l.979.261v5.081l-1.389-.074z"></path><path d="M46.283 144.732a22.475 22.475 0 1122.474-22.475 22.5 22.5 0 01-22.474 22.475zm0-42.311a19.838 19.838 0 1019.837 19.838 19.86 19.86 0 00-19.837-19.836z"></path><path d="M46.283 132.359a9.263 9.263 0 119.263-9.263 9.274 9.274 0 01-9.263 9.263zm0-15.888a6.626 6.626 0 106.626 6.626 6.631 6.631 0 00-6.626-6.626z"></path></g><path fill="none" d="M42.709 176.212h7.157"></path><path fill="#21b790" d="M42.709 174.894h7.157v2.637h-7.157z"></path><g fill="#21b790"><path d="M50.916 180.16a21.108 21.108 0 01-4.535.538 22.362 22.362 0 01-4.612-.538"></path><path d="M46.383 182.016a23.5 23.5 0 01-4.922-.575l.617-2.564a21.2 21.2 0 004.3.5 19.985 19.985 0 004.22-.5l.638 2.558a22.307 22.307 0 01-4.853.581z"></path></g><circle cx="46.287" cy="165.733" r="2.024" fill="#21b790"></circle><circle cx="46.287" cy="123.053" r="4.278" fill="#21b790"></circle><path fill="#191817" d="M13.585 208.252h261.648v2.637H13.585z"></path><path fill="#fff" d="M131.188 138.451h20.724v16.163h-20.724z"></path><path d="M153.23 155.932h-23.361v-18.8h23.361zm-20.724-2.637h18.087v-13.526h-18.087z" fill="#191817"></path><g fill="#21b790"><path d="M184.042 153.153a25.692 25.692 0 1125.69-25.692 25.719 25.719 0 01-25.69 25.692zm0-48.745a23.055 23.055 0 1023.053 23.055 23.079 23.079 0 00-23.053-23.052z"></path><path d="M184.073 144.332a16.922 16.922 0 11-.071-33.843q.308 0 .615.01l4.145-2.524 2.521 4.145a16.924 16.924 0 01-7.21 32.212zm-.059-31.207a14.289 14.289 0 105.821 1.231l-.377-.167-1.578-2.59-2.591 1.577-.41-.024a13.88 13.88 0 00-.863-.025z"></path><path d="M182.524 125.232l-2.573 5.972h-1.8l2.607-5.766h-3.051v-1.5h4.822zM186.288 126.144q0-.834-.722-.834a.663.663 0 00-.6.284 1.752 1.752 0 00-.208.886h-1.7a2.706 2.706 0 01.794-1.974 2.661 2.661 0 011.852-.662 2.414 2.414 0 011.765.59 2.072 2.072 0 01.595 1.537 3.248 3.248 0 01-.8 2.051 7.483 7.483 0 01-1.923 1.694h2.842v1.406h-5.128v-1.3q3.234-2.315 3.233-3.678z"></path><circle cx="189.572" cy="123.918" r=".867"></circle><path d="M183.529 105.089h1.026v2.901h-1.026zM167.858 112.005l.726-.725 2.05 2.051-.725.726zM197.36 113.419l2.138-2.14.726.724-2.139 2.14zM175.035 106.976l.949-.391 1.106 2.681-.948.392zM201.731 119.609l2.798-1.155.392.948-2.798 1.155zM163.187 119.346l.394-.948 2.678 1.114-.394.947zM190.997 109.4l1.163-2.794.946.393-1.162 2.795z"></path></g><path d="M167.502 95.342v-.131a25.977 25.977 0 10-51.953-.247v.247c0 .082 0 .148.005.223v19.745c0 .074-.005.147-.005.222a25.977 25.977 0 0051.953 0v-.131h.045V95.342z" fill="#fff"></path><path d="M141.529 142.697a27.326 27.326 0 01-27.3-27.3v-20.02-.166a27.3 27.3 0 0154.567-1.188h.068v22.566h-.068a27.328 27.328 0 01-27.267 26.108zm0-72.144a24.685 24.685 0 00-24.658 24.658l.005.213-.005 19.977a24.658 24.658 0 0049.316.241v-1.69h.045V96.66l-.049-1.318v-.118a24.688 24.688 0 00-24.654-24.671z" fill="#191817"></path><path d="M121.388 105.111h-7.748a1.319 1.319 0 110-2.637h7.748a1.319 1.319 0 110 2.637zM145.077 105.111h-6.84a1.319 1.319 0 110-2.637h6.84a1.319 1.319 0 110 2.637zM169.46 105.111h-8.043a1.319 1.319 0 010-2.637h8.046a1.319 1.319 0 110 2.637z" fill="#191817"></path><path d="M182.998 190.718h-82.9v-16.075s6.941-20.228 23.8-20.228h35.3c16.857 0 23.8 20.228 23.8 20.228z" fill="#fff"></path><path d="M184.316 192.032H98.783v-17.606l.071-.209a41.444 41.444 0 015.776-10.483c5.286-6.957 11.949-10.635 19.27-10.635h35.3c7.32 0 13.984 3.678 19.27 10.635a41.393 41.393 0 015.775 10.483l.071.209zm-82.895-2.637h80.262v-14.519c-.762-2.036-7.637-19.142-22.479-19.142h-35.3c-14.843 0-21.715 17.106-22.479 19.142z" fill="#191817"></path><path d="M106.405 107.326c.125-3.6 2.377-6.4 5.049-6.266h2.164c.8 0 1.442.872 1.442 1.946v15.566c0 1.075-.645 1.946-1.442 1.946h-2.164c-2.672.132-4.923-2.663-5.049-6.266z" fill="#fff"></path><path d="M111.283 121.841a5.387 5.387 0 01-3.638-1.487 8.61 8.61 0 01-2.553-6.057v-7.018a8.606 8.606 0 012.553-6.057 5.385 5.385 0 013.844-1.482h2.134a3.011 3.011 0 012.76 3.265v15.567a3.011 3.011 0 01-2.76 3.265h-2.14c-.068.002-.136.004-.2.004zm-3.554-14.493v6.9a6.019 6.019 0 001.7 4.16 2.739 2.739 0 001.965.789h2.159a.944.944 0 00.193-.627v-15.565a.945.945 0 00-.193-.628h-2.159a2.754 2.754 0 00-1.964.789 6.041 6.041 0 00-1.706 4.182z" fill="#191817"></path><path d="M176.661 114.252c-.125 3.6-2.377 6.4-5.048 6.266h-2.167c-.8 0-1.442-.871-1.442-1.946v-15.567c0-1.075.644-1.947 1.442-1.947h2.168c2.672-.131 4.923 2.663 5.049 6.266z" fill="#fff"></path><path d="M171.79 121.841h-2.344a3.012 3.012 0 01-2.761-3.265v-15.571a3.012 3.012 0 012.761-3.265h2.137a5.33 5.33 0 013.843 1.482 8.61 8.61 0 012.553 6.057v7.021a8.6 8.6 0 01-2.553 6.056 5.38 5.38 0 01-3.636 1.485zm-2.27-2.642h2.16a2.77 2.77 0 001.964-.788 6.052 6.052 0 001.7-4.185v-6.9a6.027 6.027 0 00-1.7-4.161 2.735 2.735 0 00-1.964-.789h-2.16a.946.946 0 00-.193.628v15.568a.947.947 0 00.19.627z" fill="#191817"></path><rect x="83.666" y="168.642" width="115.222" height="57.579" rx="6.9" fill="#fff"></rect><path d="M191.983 227.539H90.567a8.229 8.229 0 01-8.22-8.219v-43.776a8.23 8.23 0 018.22-8.221h101.416a8.23 8.23 0 018.22 8.22v43.776a8.229 8.229 0 01-8.22 8.22zM90.567 169.96a5.59 5.59 0 00-5.583 5.583v43.776a5.588 5.588 0 005.583 5.582h101.416a5.588 5.588 0 005.583-5.582v-43.776a5.59 5.59 0 00-5.583-5.583z" fill="#191817"></path><path d="M231.635 91.632h-2.638v-2.633h2.638zm0-5.274h-2.638v-2.633h2.638zm0-5.274h-2.638v-2.239h.922v-1.319h1.716zm-4.352-.918h-2.637v-2.634h2.637zm-5.274 0h-2.637v-2.634h2.637zm-5.274 0h-2.637v-2.634h2.637zm-5.275 0h-2.637v-2.634h2.637zm-5.274 0h-2.637v-2.634h2.634zm-5.274 0h-1.465v-3.809h2.636v2.488h-1.172zm1.172-6.446h-2.637v-2.637h2.636zm0-5.275h-2.637v-2.2h2.636z" fill="#21b790"></path><path d="M129.427 113.889c-2.878 0-6.148-.079-6.355-.083a2.44 2.44 0 01-2.508-2.339v-9.259a2.449 2.449 0 012.541-2.339h13.588a2.45 2.45 0 012.542 2.339v7.038c0 2.086-3.8 4.363-6.089 4.554-.796.069-2.202.089-3.719.089zm-6.227-2.718c.74.017 7.677.176 9.728 0a6.419 6.419 0 003.672-2.071v-6.587h-13.4zM153.747 113.932a34.522 34.522 0 01-3.764-.14c-2.324-.311-6.024-2.52-6.024-4.547v-7.038a2.45 2.45 0 012.542-2.339h13.589a2.449 2.449 0 012.541 2.339v9.259a2.448 2.448 0 01-2.541 2.338c-.178.01-3.456.128-6.343.128zm-7.15-4.839a7.036 7.036 0 003.737 2.085c1.676.225 7.5.071 9.661-.008v-8.659l-13.4-.006z" fill="#191817"></path><g fill="#21b790"><path d="M86.775 89.547h-13.4a2.492 2.492 0 01-2.492-2.492l.011-6.926h2.637l-.01 6.78H86.63l-.011-6.78h2.637l.011 6.926a2.492 2.492 0 01-2.488 2.5z"></path><path d="M80.077 92.332c-3.233 0-5.491-1.277-5.491-3.106 0-.149.007-.432.007-.432v-.376h2.637v.41c0 .022 0 .118-.005.221a6.614 6.614 0 005.7 0v-.63h2.637v.406s.007.262.007.4c-.002 1.825-2.26 3.107-5.492 3.107zM88.601 81.371h-17.05a2.709 2.709 0 01-2.708-2.709v-.006l.01-3.373a19.851 19.851 0 1122.445 0l.01 3.373a2.706 2.706 0 01-2.7 2.715zm-8.536-39.67a17.211 17.211 0 00-9.182 31.763l.615.389-.015 4.813.071.071h17.05l.069-.071-.014-4.813.616-.389a17.212 17.212 0 00-8.095-31.726q-.561-.037-1.115-.037z"></path><circle cx="80.076" cy="68.998" r="3.299"></circle><path d="M80.077 54.16a15.227 15.227 0 00-11.545 5.3l1.926 1.926a12.518 12.518 0 0119.239 0l1.926-1.926a15.224 15.224 0 00-11.546-5.3z"></path><path d="M80.077 59.581a9.858 9.858 0 00-7.707 3.716l1.938 1.935a7.137 7.137 0 0111.538 0l1.937-1.935a9.856 9.856 0 00-7.706-3.716zM72.201 82.82H87.95v2.637H72.201z"></path></g><g><path d="M142.049 79.446s5.9-3.634 7.208-18.733-10.724-20.8-10.724-20.8 1.061 19.675-5.5 21.593" fill="#fff"></path><path d="M142.741 80.569l-1.4-2.237c.054-.035 5.383-3.658 6.6-17.732.9-10.431-4.857-16.035-8.018-18.3.067 5.7-.459 18.694-6.522 20.467l-.74-2.531c2.095-.612 3.61-4.293 4.267-10.363a70.306 70.306 0 00.287-9.9l-.117-2.21 2 .948a22.376 22.376 0 0111.472 22.121c-1.345 15.578-7.566 19.575-7.829 19.737z" fill="#191817"></path></g><g><path d="M130.555 66.425c-8.347-10.621-21.425-6.311-21.425-6.311s13.065 12.006 9.053 17.606" fill="#fff"></path><path d="M119.255 78.488l-2.144-1.535c1.252-1.748.191-5.058-2.989-9.321a54.063 54.063 0 00-5.886-6.547l-1.652-1.52 2.132-.7c.139-.047 14.059-4.471 22.874 6.748l-2.073 1.629c-5.687-7.237-13.83-7-17.727-6.374 3.693 3.769 11.171 12.447 7.465 17.62z" fill="#191817"></path></g><g><path d="M140.056 76.22s-2.245-6.681 7.069-16.723S170 55.206 170 55.206s-14.35 10.984-10.4 17.137" fill="#fff"></path><path d="M138.806 76.64c-.1-.3-2.383-7.541 7.352-18.039 9.83-10.6 24.213-4.67 24.357-4.609l2.084.885-1.8 1.376c-3.472 2.66-12.778 11.193-10.094 15.377l-2.222 1.425c-3.654-5.694 4.755-13.889 8.817-17.356-4.01-1.089-12.7-2.328-19.214 4.7-8.564 9.235-6.8 15.36-6.781 15.42z" fill="#191817"></path></g><g><path d="M120.392 69.712s-12.315 4.391-9.133 25.361c.235 1.546 3.883 6.427 3.883 6.427s3.532-6.109 4.963-9.451c-.381-3.149.191-6.3 5.157-8.591s16.674-1.875 21-3.15 4.582-6.492 4.582-6.492 8.5 1.655 10.787 9.547.477 9.451.477 9.451 4.391 10.532 4.391 10.021 2.919-5.759 3.437-9.037c.955-6.047-1.816-15.593-1.816-15.593s-.987-6.427 3.34-10.882l4.328-4.454s-19.7-4.044-26.38 3.309c-3.525 3.88-12.406-5.346-29.016 3.534z" fill="#fff"></path><path d="M166.317 104.208c-.886 0-1.41-1.255-5.426-10.888l-.368-.883.635-.546c.143-.264.956-2.138-.794-8.161-1.534-5.288-6.153-7.471-8.453-8.248a7.865 7.865 0 01-5.274 6.091 48.346 48.346 0 01-8.388 1.071c-4.56.358-9.728.761-12.433 2.011-4.146 1.914-4.757 4.278-4.4 7.236l.043.352-.14.326c-1.446 3.375-4.888 9.339-5.034 9.591l-1 1.731-1.2-1.6c-.9-1.208-3.868-5.281-4.13-7.02-3.19-21.019 8.562-26.243 9.89-26.762 10.671-5.677 18.3-4.167 23.348-3.164 2.894.574 4.44.832 5.242-.051 7.106-7.816 26.785-3.885 27.62-3.713l2.358.483-6.01 6.181c-3.572 3.676-3.073 9.016-2.993 9.7.312 1.1 2.769 10.066 1.822 16.066a33.426 33.426 0 01-2.795 7.6c-.247.536-.527 1.135-.626 1.4v.913l-1.066.229a2.374 2.374 0 01-.428.055zm-2.734-11.276a517.265 517.265 0 002.857 6.72 27.818 27.818 0 002.19-6.057c.9-5.681-1.751-14.927-1.778-15.019l-.036-.167c-.045-.289-1.032-7.132 3.7-12l2.575-2.65c-5.328-.772-17.928-1.944-22.7 3.31-1.842 2.026-4.43 1.514-7.706.864-4.976-.986-11.789-2.336-21.668 2.944l-.176.078c-.49.182-11.236 4.416-8.276 23.923a19.506 19.506 0 002.447 4.183c1.09-1.949 2.8-5.078 3.746-7.214-.288-2.912.156-6.908 5.952-9.583 3.133-1.446 8.551-1.871 13.331-2.246a48.574 48.574 0 007.853-.972c3.357-.987 3.628-5.12 3.637-5.3l.085-1.509 1.484.283c.379.074 9.318 1.907 11.8 10.473 1.702 5.873 1.272 8.662.683 9.939z" fill="#191817"></path></g><g><path d="M121.442 75.632s11.264-16.705-.255-28.224c26.7 2.068 29.657 25.074 29.657 25.074" fill="#fff"></path><path d="M121.442 76.949a1.319 1.319 0 01-1.094-2.055 32.81 32.81 0 004.023-9.441c1.682-6.944.3-12.7-4.116-17.115l-2.52-2.52 3.557.275c27.448 2.127 30.831 25.98 30.862 26.221a1.318 1.318 0 01-2.615.337c-.113-.855-2.918-20.133-25.206-23.559 8.37 11.9-1.685 27.112-1.8 27.276a1.32 1.32 0 01-1.091.581z" fill="#191817"></path></g><path d="M138.054 82.199a1.318 1.318 0 01-.719-2.425c4.326-2.8 3.065-9.45 3.052-9.517a1.318 1.318 0 112.585-.521c.07.344 1.641 8.466-4.2 12.248a1.3 1.3 0 01-.718.215zM156.955 77.232c-.044 0-.088 0-.133-.006a1.319 1.319 0 01-1.18-1.444c.629-6.3 4.637-8.894 4.8-9a1.32 1.32 0 111.415 2.227c-.146.095-3.106 2.1-3.6 7.039a1.318 1.318 0 01-1.302 1.184z" fill="#191817"></path><circle cx="141.553" cy="197.534" r="8.401" fill="#26dcf3"></circle><path d="M186.401 96.715h-2.637v-2.637h2.637zm0-5.274h-2.637v-2.637h2.637zm0-5.274h-2.637v-2.635h2.637zm0-5.274h-2.637v-2.638h2.637zm0-5.275h-2.637v-2.637h2.637zm0-5.274h-2.637v-2.637h2.637zM97.368 108.744h-2.637v-2.638h2.637zm0-5.275h-2.637v-2.637h2.637zm0-5.274h-2.637v-2.637h2.637zm0-5.274h-2.637v-2.637h2.637zm0-5.274h-2.637v-2.638h2.637zm-3.837-1.439h-2.637v-2.637h2.637z" fill="#21b790"></path></svg>',
    presentationWidth: 200,
  },
}
