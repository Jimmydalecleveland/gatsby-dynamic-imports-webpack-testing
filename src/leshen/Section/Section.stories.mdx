import { Story, Meta, Preview, Props, Anchor } from '@storybook/addon-docs/blocks'

import Section, { ticketID } from '.'
import ComponentProgress from '../ComponentProgress'
import Typography from '../Typography'
import SectionImage from '../../images/contentful-stories/section.png'
import Link from '../Link'
import Button from '../Button'
import { SVG_IMAGE_DATA } from './dummyData'
import Img from '../Image'

<Meta title='Components|Section' component={Section} decorators={[(story) => <><ComponentProgress ticketID={ticketID} />{story()}</>]} />

# Section
<Preview>
  <Story name="with h1">
    <Section>
      <Typography variant="h1">A Section adds the default padding of the theme.</Typography>
         <Typography>
            CenturyLink Price for Life Internet Plans ensure your monthly
            internet rate stays the same when you remain on the same internet
            plan at the same address. No contract, no rate hikes.
          </Typography>
          <Typography>
            See what CenturyLink Services are in your area: <Link to="/section-link"> Call Now</Link>
          </Typography>
          <Button primary>Submit</Button>
    </Section>
  </Story>
</Preview>

## Props
<Props of={Section} />

# Section
<Preview>
  <Story name="with links and buttons">
    <Section>
       <Typography variant="h3">A Section adds the default padding of the theme.</Typography>
       <Typography>
            CenturyLink Price for Life Internet Plans ensure your monthly
            internet rate stays the same when you remain on the same internet
            plan at the same address. No contract, no rate hikes.
          </Typography>
          <Typography>
            See what CenturyLink Services are in your area: <Link to="/section-link"> Call Now</Link>
          </Typography>
          <Button primary>Submit</Button>
    </Section>
  </Story>
</Preview>

# Section
<Preview>
  <Story name="Prominent background with links and buttons">
    <Section backgroundColor="Prominent">
       <Typography variant="h3">A Section adds the default padding of the theme.</Typography>
       <Typography>
            CenturyLink Price for Life Internet Plans ensure your monthly
            internet rate stays the same when you remain on the same internet
            plan at the same address. No contract, no rate hikes.
          </Typography>
          <Typography>
            See what CenturyLink Services are in your area: <Link to="/section-link"> Call Now</Link>
          </Typography>
          <Button primary>Submit</Button>
    </Section>
  </Story>
</Preview>

## Contentful
<Preview>
  <Story name="contentful">
    <>
      <Typography>Add "Sections" at the bottom of a page.</Typography>
      <img src={SectionImage} alt="Section" />
    </>
  </Story>
</Preview>


## Section with image

<Preview>
  <Story name="SVG image at top">
    <Section backgroundColor="Prominent">
      <Img data={SVG_IMAGE_DATA} />
       <Typography variant="h3">A Section adds the default padding of the theme.</Typography>
       <Typography>
            CenturyLink Price for Life Internet Plans ensure your monthly
            internet rate stays the same when you remain on the same internet
            plan at the same address. No contract, no rate hikes.
          </Typography>
          <Typography>
            See what CenturyLink Services are in your area: <Link to="/section-link"> Call Now</Link>
          </Typography>
          <Button primary>Submit</Button>
    </Section>
  </Story>
</Preview>

