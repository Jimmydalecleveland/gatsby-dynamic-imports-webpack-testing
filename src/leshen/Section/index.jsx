import React from 'react'
import PropTypes from 'prop-types'

import * as Styled from './Section.styles'

const Section = ({ children, ...props }) => (
  <Styled.Section {...props}>{children}</Styled.Section>
)

Section.defaultProps = {
  children: null,
}

Section.propTypes = {
  children: PropTypes.oneOfType([
    PropTypes.node,
    PropTypes.objectOf(PropTypes.node),
    PropTypes.arrayOf(PropTypes.node),
  ]),
}

export default Section

export const ticketID = 'CMS-174'
