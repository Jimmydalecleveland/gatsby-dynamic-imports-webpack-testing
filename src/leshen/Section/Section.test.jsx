import React from 'react'
import { render } from 'setup-tests'
import Section from '.'

describe('Section', () => {
  test('Section should render', () => {
    const { container } = render(
      <Section
        heading="Heading"
        subheading="Sub Heading"
        mainContent={<p>Child</p>}
      />
    )
    expect(container.firstChild).toMatchSnapshot()
  })
})
