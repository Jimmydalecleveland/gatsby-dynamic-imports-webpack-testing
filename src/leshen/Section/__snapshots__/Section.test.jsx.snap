// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Section Section should render 1`] = `
.emotion-0 {
  padding-left: calc(2.2222222222222223vw + 21.333333333333332px);
  padding-right: calc(2.2222222222222223vw + 21.333333333333332px);
  padding-top: calc(7.777777777777778vw + -5.333333333333333px);
  padding-bottom: calc(7.777777777777778vw + -5.333333333333333px);
  position: relative;
}

@media (max-width:480px) {
  .emotion-0 {
    padding-left: 32px;
  }
}

@media (min-width:1200px) {
  .emotion-0 {
    padding-left: 48px;
  }
}

@media (max-width:480px) {
  .emotion-0 {
    padding-right: 32px;
  }
}

@media (min-width:1200px) {
  .emotion-0 {
    padding-right: 48px;
  }
}

@media (min-width:1288px) {
  .emotion-0 {
    padding-right: calc((100% - 1200px) / 2);
    padding-left: calc((100% - 1200px) / 2);
  }
}

@media (max-width:480px) {
  .emotion-0 {
    padding-top: 32px;
  }
}

@media (min-width:1200px) {
  .emotion-0 {
    padding-top: 88px;
  }
}

@media (max-width:480px) {
  .emotion-0 {
    padding-bottom: 32px;
  }
}

@media (min-width:1200px) {
  .emotion-0 {
    padding-bottom: 88px;
  }
}

<section
  class="emotion-0 emotion-1"
/>
`;
